README.md

```diff
@@ -22,18 +22,22 @@ docker run -p 19120:19120 projectnessie/nessie
 
 A local [Web UI](https://projectnessie.org/tools/ui/) will be available at this point.
 
-Then install the Nessie CLI tool
+Then install the Nessie CLI tool (to learn more about CLI tool and how to use it, check [here](https://projectnessie.org/tools/cli/)
```

Comment:
nit: `here` -> `Project Nessie Documentation` ?

Reply:
ok. changed it to `Nessie CLI Documentation`